app/content/pages/community/jobs.html.erb

```diff
@@ -5,7 +5,7 @@ title: Jobs
 <%= render Page::ContainerComponent.new(page: current_page) do |page| %>
   <% page.with_title(title: current_page.data.fetch("title")) %>
 
-  https://railshotwirejobs.com/jobs<br>
-  https://rubyonremote.com/remote-hotwire-jobs/<br>
-  https://gorails.com/jobs<br>
+  <a href="https://railshotwirejobs.com/jobs">https://railshotwirejobs.com/jobs</a><br>
```

Comment:
Hey @redbar0n, thank you! 🙏🏼 

Would you mind converting the links to `link_to` calls and using `target="_blank"` so it's consistent across all pages.
```suggestion
  <%= link_to "https://railshotwirejobs.com/jobs", "https://railshotwirejobs.com/jobs", target: :_blank %><br>
```